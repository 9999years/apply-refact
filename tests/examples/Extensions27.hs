{-# LANGUAGE DeriveGeneric, GeneralizedNewtypeDeriving #-} 
newtype Micro = Micro Int deriving Generic