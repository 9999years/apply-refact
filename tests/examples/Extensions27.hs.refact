[("tests/examples/Extensions27.hs:1:1-58: Warning: Unused LANGUAGE pragma\nFound:\n  {-# LANGUAGE DeriveGeneric, GeneralizedNewtypeDeriving #-}\nPerhaps:\n  {-# LANGUAGE DeriveGeneric #-}\nNote: Extension GeneralizedNewtypeDeriving is not used\n",[ModifyComment {pos = SrcSpan {startLine = 1, startCol = 1, endLine = 1, endCol = 59}, newComment = "{-# LANGUAGE DeriveGeneric #-}"}])]
