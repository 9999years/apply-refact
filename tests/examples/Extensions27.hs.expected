{-# LANGUAGE DeriveGeneric #-}
newtype Micro = Micro Int deriving Generic
